src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -236,4 +240,77 @@ private static boolean checkIsAllLinesAreWhitespace(String[] lines, int lineFrom
         }
         return result;
     }
+
+    /**
+     *  Checks for empty LITERAL_CASE.
+     *
+     *  @param ast a {@code DetailAST} value
+     */
+    protected void checkForEmptyCase(DetailAST ast) {
+        final DetailAST slistToken = ast.getNextSibling();
+        if (slistToken != null
+                && slistToken.getType() == TokenTypes.SLIST) {
+            final boolean emptyCase;
```

Comment:
this variable is not required here, please move it inside blocks .... or completely get rid of it.